### PR TITLE
[RLlib] Fix minor typos in docs

### PR DESCRIPTION
**Pull request by lucasalavapena**
## Why are these changes needed?

There were two small typos in the RLlib docs.

## Related issue number

NA

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by zhe-thoughts — APPROVED** [review]
Thanks for the contribution @lucasalavapena ! Let's wait for a committer review / merge

**Review by dynamicwebpaige — APPROVED** [review]
Looks awesome to me, and thank you for making the fixes! 🙌

**Pull request merged**
